app/public/dist/client/locales/en/translation.json

```diff
@@ -1256,383 +1256,383 @@
-    "CROSS_POISON": "Deal [30,60,120,SP] SPECIAL to target and enemies adjacent to it, and POISONNED them for 3 seconds",
+    "AURASPHERE": "Exude an aura and project energy in a line, dealing [25,50,SP] SPECIAL and inflicting all enemy Pokémon hit with SILENCE for 3 seconds.",
+    "SKETCH": "Copy the ability, max PP and STAR of the allied Pokémon on the user's left at the start of the fight.",
+    "OVERDRIVE": "Twang an electric guitar, causing a huge echo and strong vibration. Deal [120,SP]% of ATK as SPECIAL within a 3 tile radius of the user, -20% per tile from the user.",
+    "LOVELY_KISS": "Kiss the enemy Pokémon with a frightening expression. Put the target to SLEEP for [2,4,6,SP] seconds. If the target is already suffering from SLEEP, deal [50,100,150,SP] SPECIAL instead.",
+    "TRANSFORM": "Take the appearance and the stats of the target (except HP).",
+    "PSYCHIC_FANGS": "Deal [100,SP] PHYSICAL and remove all the target's ATK, DEF and SPE_DEF stat increases.",
+    "SHED_TAIL": "Create a substitute and charge towards the ally with the lowest HP, granting them [80, SP] SHIELD.",
+    "NIGHT_SLASH": "Deal [15,30,60,SP] SPECIAL to the target. Can crit by default.",
+    "KOWTOW_CLEAVE": "Deal 150% ATK to the target, + [20,SP]% TRUE for each fallen allied Pokémon. Can crit by default",
+    "SHIELDS_DOWN": "Transform Minior to a random core form.",
+    "SHIELDS_UP": "Transform Minior to its meteor form.",
+    "SANDSEAR_STORM": "Searingly hot sand that strikes all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and applying BURN for 2 seconds.",
+    "WILDBOLT_STORM": "Thunderous tempest that strikes all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and inflicting PARALYSIS for 4 seconds.",
+    "BLEAKWIND_STORM": "Savagely cold winds that strike all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and applying FREEZE for 2 seconds.",
+    "SPRINGTIDE_STORM": "Fierce winds brimming with love and hate strike all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and inflicting CHARM for 2 seconds.",
+    "AURA_WHEEL": "Deal [60,SP] SPECIAL to the target, gain [10,SP]% ATK_SPEED and switch between Normal and Hangry form.",
+    "LICK": "Deal [30,60,120,SP] SPECIAL to the target and inflict them with CONFUSION and PARALYSIS for 3 seconds.",
+    "FURY_SWIPES": "Deal 100% of ATK as PHYSICAL to the target [2,SP] to [5,SP] times.",
+    "TICKLE": "Reduce ATK and DEF of the [1,2] closest enemy Pokémon by [3,SP].",
+    "AROMATHERAPY": "Cure negative status Conditions and heal adjacent allied Pokémon for [20,40,80,SP] HP.",
+    "DETECT": "PROTECT adjacent allied Pokémon for [0.5,SP] seconds, multiplied by the number of enemy Pokémon detected in a 2 tile radius.",
+    "SPACIAL_REND": "Deal [100,SP] SPECIAL to all enemy Pokémon in the same row as the target and reverse the position of the enemy team.",
```

Comment:
the original order is better, "then" clarifies that the damage is done AFTER the position of enemies has changed

Reply:
This order is more consistent with other abilities. If "and" is changed to "after" it should be ok right?

Reply:
reversing the position is the strongest and more unique effect, it should come first

Reply:
Fixed. I'll probably reorder all of the ability descriptions so that unique effects are listed first in the future.